### PR TITLE
adding tooltip property to the controlBar configuration

### PR DESCRIPTION
**Pull request by FernandoCuevas**

### CODE REVIEW

File: languageFiles/es.json [hunk 1]
```diff
@@ -112,5 +112,12 @@
   "Uniform": "Uniforme",
   "Depressed": "En relieve",
   "Raised": "Elevado",
-  "Shadow": "Sombra"
+  "Shadow": "Sombra",
+  "Play": "Reproducir",
+  "Pause": "Pausa",
+  "Mute": "Silenciar",
+  "Unmute": "Activar audio",
+  "Closed Captions": "Subtitulos",
```
Comment (FernandoCuevas) [inline languageFiles/es.json]: I'm not sure if 'Subtitulos' is the correct translation for 'Closed captions'

File: languageFiles/es.json [hunk 2]
```diff
@@ -9,7 +9,7 @@
   "CC Options": "Opciones de subtitulado",
   "On": "Sí",
   "Off": "No",
-  "Discover": "Descubrir",
+  "Discovery": "Descubrir",
```
Comment (pilievOoyala) [inline languageFiles/es.json]: Can you make the same change to the english file? It's also 'Discover'

**Comment by aeng7** [comment]
LGTM

File: languageFiles/es.json (hunk 2 above)
Comment (aeng7) [inline languageFiles/es.json]: Is it possible this was meant to be Discover? Who can we check this with?

Comment (aeng7) [inline languageFiles/es.json]: Steve says this should be left as is as Discover. This is assuming this is the string at the top of the discovery UI:

screen shot 2017-08-09 at 4 34 22 pm

Comment (FernandoCuevas) [inline languageFiles/es.json]: Modified to Discovery because that's how it's named in the english language file. Could you please check that ?

Comment (pilievOoyala) [inline languageFiles/es.json]: @FernandoCuevas  You are right. I think we misread it.

**Comment by pilievOoyala** [comment]
LGTM

File: languageFiles/es.json (hunk 2 above)
Comment (FernandoCuevas) [inline languageFiles/es.json]: never mind, found another 'Discover' constant. I'll revert this change.

**Comment by pilievOoyala** [comment]
@FernandoCuevas  We still need to revert discovery back to discover in the spanish file.

**Pull request merged**